music21/musicxml/m21ToXml.py

```diff
@@ -3099,7 +3099,7 @@ def objectAttachedSpannersToNotations(self, obj, objectSpannerBundle=None):
 
         return notations
 
-    def noteToXml(self, n, addChordTag=False, chordParent=None):
+    def noteToXml(self, n, pitchNumber=0, chordParent=None):
```

Comment:
pitchNumber is too vague as a variable name -- does it mean C=0, C#=1? is it midi number?  it might be "noteIndexInChord" or something like that